js/ooyala_ssai.js

```diff
@@ -337,6 +339,11 @@ OO.Ads.manager(function(_, $)
      */
     this.resumeAd = function(ad)
     {
+      //Setting the ad callback again since ad was resumed
+      this.adIdDictionary[ad.ad.data.id].adTimer = _.delay(
```

Comment:
Can we add null checks for the following?
ad
ad.ad
ad.ad.data
this.adIdDictionary[ad.ad.data.id]
currentId3Object
?